pandas/core/arrays/datetimelike.py

```diff
@@ -1816,7 +1816,7 @@ def validate_endpoints(closed):
 
     Parameters
     ----------
-    closed : {None, "left", "right"}
+    closed : {None, False, True, "neither", "left", "right", "both"}
```

Comment:
If we move towards standardisation is it better to document the argument inputs as `{"neither", "both", "left", "right"}` including some notes that `True` `False` and `None` remain accepted for backwards compatability?

Reply:
That does sound better, but where do you recommend putting the notes that `True`, `False`, and `None` remain accepted?